File: BadReadsAPIApp/BadReadsAPI/Comment.cs
using System.ComponentModel.DataAnnotations;

namespace BadReadsAPI
{
    public class Comment
    {
        public int Id { get; set; }
        [StringLength(200)]
        public string Description { get; set; } = string.Empty;
        public float Rating { get; set; }
        public int UserId { get; set; }
        public User User { get; set; }
    }
}

File: BadReadsUIApp/src/app/models/comment.ts
import { User } from "./user";

export class Comment {

    public id: number;
    public description: string;
    public rating: number;
    public user: User;

    constructor (
        id: number,
        description: string,
        rating: number,
        user: User
    ) {
        this.id = id;
        this.description = description;
        this.rating = rating;
        this.user = user;
    }

    fromJson(json: any) {
        return new Comment(
            json.id,
            json.description,
            json.rating,
            json.user
        )
    }

}